script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_1B.txt

```diff
@@ -24,29 +24,29 @@ I silently take a sip of the dark red tea and―――# wait, what's this taste.
 "It's so delicious...
 It's bitter yet sweet, complex yet cleansing on the throat, and it smells just like Christmas...!"
 <Page7>
-I don't know what I'm saying, but that's how shocked I am.
+I don't know what I'm saying, but that's just how shocked I am.
 I learned last night of the grandeur of the Tohno residence, and this tea is no exception.
 <Page8>
 "This morning we are having Esprit de Noel from Mariage.
-The most famous tea stores are in England, but this is from France. It was founded around 1850 and only recently landed in Japan about 15 years ago... so it is not a major brand, but the quality is good."
+The most famous tea stores are in England, but this is from France. It was founded around 1850 and only recently landed in Japan about 15 years ago... It is not a major brand, but the quality is good." //New sentence after ellipsis?
 <Page9>
-"Fufu...the expression Christmas is so vague but strangely enough, it conveys the nuance."
+"Fufu... The expression 'Christmas' is so vague but strangely enough, it conveys the nuance."
 I'm not sure if my expression was too funny, but Akiha is smiling with pride.
 <Page10>
 "I hear the price is fair, and it seems to fit the taste of my commoner brother quite well. Did you know that a fine leaf is worth its weight in gold?"
 <Page11>
-It was the condescending attitude of a ruler, looking down upon a pathetic, primitive man.
+It is the condescending attitude of a ruler, looking down upon a pathetic, primitive man. //Does "is" or "was" fit better here?
 Even her gesture when placing the teacup on the saucer is elegant.
 <Page12>
 "Tch..."
 But, unfortunately, she's right.
-This tea is the real deal... what was the tea I've been drinking in the afternoon until now? I mean, you could certainly start a business or two with this. If such a thing existed in a time period where entertainment was scarce, even I would be ordering this in desperation.
+This tea is the real deal... What was the tea I've been drinking in the afternoon until now? I mean, you could certainly start a trade company or two with this. If such a thing existed in a time period where entertainment was scarce, even I would be ordering this in desperation. //Pretty sure Shiki is referencing the East India Trading Company and similar groups here, not just businesses in general. Tea so good it justifies colonialism...
```

Comment:
Hmm, I think that even if this is the implication we should let the reader work it out themselves here.